tests/stdlib/tfenv.nim

```diff
@@ -1,8 +1,7 @@
-import fenv
+import std/fenv
 
 
 func is_significant(x: float): bool =
-  if x > minimumPositiveValue(float) and x < maximumPositiveValue(float): true
-  else: false
+  x > minimumPositiveValue(float) and x < maximumPositiveValue(float)
```

Comment:
@konsumlamm shouldn't this be:
```nim
  x >= minimumPositiveValue(float) and x <= maximumPositiveValue(float)
```
followup PR welcome

Reply:
Hmm... probably. Would it make sense to add more tests? I'm not sure, since the vars/procs/templates in this module are just C bindings and constants.

Reply:
> Would it make sense to add more tests? 

yes, bindings could be wrong, or require certain flags (eg: -lm) that aren't set on some configuration, etc; plus, having a test makes it easier for user code to use those procs, tests for the most tricky to use procs (proc fe*...) would be good; see https://github.com/timotheecour/Nim/issues/522